src/babel/util.js

```diff
@@ -109,6 +120,12 @@ export function booleanify(val: any) {
   return val;
 }
 
+function testIgnore(ignore, filename) {
+  if (ignore instanceof Function) {
+    if (ignore(filename)) return true;
+  } else if (ignore.test(filename)) return true;
```

Comment:
I tried preserving the style of the original code. I will make it a block statement. Thanks.